src/arduino-nano-element.stories.ts

```diff
@@ -0,0 +1,23 @@
+import { boolean, withKnobs } from '@storybook/addon-knobs';
+import { storiesOf } from '@storybook/web-components';
+import { html } from 'lit-html';
+import { logEvent } from 'storybook-events-logger';
+import './arduino-nano-element';
+
+storiesOf('Arduino Nano', module)
+  .addParameters({ component: 'wokwi-arduino-nano' })
+  .addDecorator(withKnobs)
+  .add(
+    'Nano',
+    () => html`
+      <wokwi-arduino-nano
+        .led13=${boolean('LED 13', false)}
+        .ledTX=${boolean('TX LED', false)}
+        .ledRX=${boolean('RX LED', false)}
+        .ledPower=${boolean('Power LED', true)}
+        .resetPressed=${boolean('Reset button', false)}
```

Comment:
We can leave this property (`resetPressed`) out of the story (it's not something the users of this element should usually change)